recipes/larray-editor/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "larray-editor" %}
+{% set version = "0.26.1" %}
+{% set sha256 = "57aafcb53fdeafe32b61d2a52185926706c048df028fbf76ecb7a86f76f7fb49" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/larray-project/larray-editor/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - larray
+    - pytest-runner
+
+  run:
+    - python
+    - larray
+    - matplotlib
+    - pyqt
+    - qtpy
+    - pytest-runner
+
+test:
+  imports:
+    - larray_editor
+
+about:
+  home: https://github.com/larray-project/larray-editor
+  license: GPL-3.0
+  license_family: GPL
+  license_file: LICENSE
```

Comment:
I'm a little surprised this works...I thought the path to the recipe had to be included, but maybe not:
https://conda-forge.org/docs/meta.html#packaging-the-license-manually